Source/WebCore/Scripts/tests/DocumentSyncData.cpp

```diff
@@ -49,4 +50,19 @@ void DocumentSyncData::update(const ProcessSyncData& data)
     }
 }
 
-} //namespace WebCore
+DocumentSyncData::DocumentSyncData(
+    bool userDidInteractWithPage,
+#if ENABLE(DOM_AUDIO_SESSION)
+    WebCore::DOMAudioSessionType audioSessionType,
```

Comment:
If you put the comma at the beginning of the line, you'll be able to have a condition on the last element too.

Reply:
OoOoOooo.

Reply:
This fixed a lot of it, but GeneratedSerializers still doesn't handle that case properly.
I'm gonna drop in a FIXME and move on with it.